app/src/main/res/values/strings.xml

```diff
@@ -566,26 +566,26 @@
     <string name="dc_custom_block_desc">Coming soon</string>
 
     <string name="dc_dns_website_heading">Show website icon in DNS logs (experimental)</string>
-    <string name="dc_dns_website_desc">Fetches website icons from <i>duckduckgo.com</i>. Consumes up to 350 mb in disk space</string>
+    <string name="dc_dns_website_desc">Fetches website icons from <i>duckduckgo.com</i>. Consumes up to 350 MB of storage space.</string>
 
     <string name="dc_dns_leaks_heading">Prevent DNS leaks</string>
-    <string name="dc_dns_leaks_desc">When enabled, RethinkDNS captures all packets on port 53 and forwards it to a user-set DNS endpoint</string>
+    <string name="dc_dns_leaks_desc">Makes RethinkDNS capture all packets on port 53 and forward them to a user-set DNS endpoint.</string>
 
     <string name="dc_check_update_heading">Prompt on blocklist updates</string>
-    <string name="dc_check_update_desc">Check for blocklist updates once every <i>three</i> days</string>
+    <string name="dc_check_update_desc">Check for blocklist updates every <i>three</i> days</string>
 
     <!-- About fragment -->
-    <string name="about_bravedns_explantion">RethinkDNS is the easiest way to monitor app activity, circumvent Internet censorship, and firewall apps on your Android device. </string>
+    <string name="about_bravedns_explantion">RethinkDNS lets you monitor app activity, circumvent Internet censorship, and firewall apps on your Android device.</string>
     <string name="about_bravedns_whoarewe">RethinkDNS is a free and open-source project, led by ex-engineers from Amazon, IBM, and Scientific Games.</string>
```

Comment:
```suggestion
    <string name="about_bravedns_whoarewe">A libre-software project, led by ex-engineers from Amazon, IBM, and Scientific Games.\nLicensed Apache 2.0—Use, study, change, and share it.</string>
```

Reply:
This sentence isn't well-formed. Let's stick to the previous one. It is just enough descriptive and not too nerdy.